### DISCUSSION TITLE
Manual backups work, but scheduled backups fail?

### DISCUSSION THREAD

**Discussion by amorris28** [borgbase/vorta#1438] [General]
Hey, y'all.

I setup Vorta to do backups to borgbase and a local hard drive so I have two profiles: one that links to the local drive and one that links to the borgbase repo. When I run these backups manually by clicking "Start Backup" they both work fine. Now, I added a scheduled backup in the Schedule tab and clicked "Apply." I get a notification from Vorta when the scheduled backup arrives, but when I look in the Archives tab, there are no new Archives for the Date/Time of the scheduled backup. Under logs at the time of the scheduled backup, for the local backup it says Category: `borg-run`, Subcommand: `list`, Returncode: `2`. For the ssh backup it says the same but Returncode: `0`.

Any idea how to troubleshoot this?
Thank you!

**Comment by real-yfprojects** [comment]
Please open a bug report! That would ease troubleshooting.
